certified-connectors/DomainToolsIrisInvestigate/apiDefinition.swagger.json

```diff
@@ -778,14 +823,14 @@
                                                 "properties": {
                                                     "value": {
                                                         "type": "string",
-                                                        "description": "Admin Contact Org",
+                                                        "description": "Organization of the administrative contact",
                                                         "title": "Admin Contact Org"

```

Comment:
@RamboV please make sure that the sentences are complete and are grammatically correct.
Make sure that English language strings are in complete form.